app/src/main/java/org/schabi/newpipe/streams/Mp4FromDashWriter.java

```diff
@@ -711,7 +712,8 @@ private int makeMoov(final int[] defaultMediaTime, final TablesInfo[] tablesInfo
 
         for (int i = 0; i < tracks.length; i++) {
             if (tracks[i].trak.tkhd.matrix.length != 36) {
-                throw new RuntimeException("bad track matrix length (expected 36) in track n°" + i);
+                throw new RuntimeException("bad track matrix length (expected 36) in"
+                        + " track n°" + i);
```

Comment:
Why split it into 2 lines? That line isn't even too long…

Reply:
My guess is that his build also fails like mine, because his system also sees it as 101 characters in a single line (because of the °, being recognized as 2 characters).

Reply:
@XiangRongLin: Then you have weird encoding settings and should fix that, not this line. That also still doesn't explain the other changes in this file.

Reply:
@XiangRongLin  nailed it. Maybe something wrong with my Android Studio, I will return it back soon.

Reply:
@ludugz: Could you also try whether #3460 fixes this?

Reply:
@wb9688 It worked. Thank you!